plugins/reolink/src/reolink-api.ts

```diff
@@ -148,6 +169,37 @@ export class ReolinkCameraClient {
         };
     }
 
+    async getAbilityWithToken() {
```

Comment:
rename to getAbility. call will always use token.

Reply:
You mean replace with the already existing getAbility?